configuration/authentication.mdx

```diff
@@ -111,13 +109,54 @@ Flipt has been tested with each of the following providers:
 - [Auth0](https://auth0.com/docs/get-started/applications/application-settings)
 - [Gitlab](https://docs.gitlab.com/ee/integration/openid_connect_provider.html)
 - [Dex](https://dexidp.io/docs/openid-connect/)
+- [Okta](https://developer.okta.com/docs/concepts/oauth-openid/#oauth-2-0)
 
 Though the intention is that it should work with other OIDC providers, these are just the handful the Flipt team has validated.
 
 Following any of the links above should take you to the relevant documentation for each of these providers' OIDC client setups.
 You can use the credentials and client configuration obtained using those steps as configuration for your Flipt instance.
 
-**Example: OIDC with Google**
+#### Callback URL
+
+When configuring your OIDC provider, you will need to provide a callback URL for the provider to redirect back to Flipt after a successful login.
+
+The callback URL will be in the form of `https://your.flipt.instance.url.com/auth/v1/method/oidc/{provider}/callback`.
+
+You can find the callback URL for each provider that you configure in your Flipt instance by querying the API.
+
+```bash
+curl --request GET \
+  --url https://your.flipt.instance.url.com/auth/v1/method \
+  --header 'Accept: application/json'
+```
+
+```json
+{
+  "methods": [
+    {
+      "method": "METHOD_TOKEN",
+      "enabled": true,
+      "sessionCompatible": false,
+      "metadata": null
+    },
+    {
+      "method": "METHOD_OIDC",
+      "enabled": true,
+      "sessionCompatible": true,
+      "metadata": {
+        "providers": {
+          "google": {
+            "authorize_url": "/auth/v1/method/oidc/google/authorize",
+            "callback_url": "/auth/v1/method/oidc/google/callback"
```

Comment:
Makes me wonder if we should actually put the `redirect_address` part in this response too (in Flipt I mean). So that these are actual URLs and not just paths.